SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -73,7 +71,7 @@ struct FConnectionConfig
 	bool EnableProtocolLoggingAtStartup;
 	FString ProtocolLoggingPrefix;
 	Worker_NetworkConnectionType LinkProtocol;
-	Worker_ConnectionParameters ConnectionParams;
+	Worker_ConnectionParameters ConnectionParams = {};
```

Comment:
Require the { }?

Reply:
Rider correctly notified me that this parameter isn't initialized in the constructor (really useful lint), so it's being zeroed now.

Reply:
Oooh nice.